docs/security/annotations.md

```diff
@@ -0,0 +1,83 @@
+# Security Annotations
```

Comment:
What is the scope of this annotations? Only for uberfire components ? (perspectives, screens etc.)

If so, we should mention this here.